cellcounter/cc_kapi/static/js/counter.js

```diff
@@ -635,19 +637,23 @@ function load_keyboard(keyboard_id) {
     }
 }
 
-function set_keyboard_primary(keyboard_id) {
+function set_keyboard_primary(keyboard_href) {
```

Comment:
Done in https://github.com/cellcounter/cellcounter/pull/446/commits/b58f420edcde591893b75b5bd43b9b77939b5708